lexos/static/js/scripts_manage.js

```diff
@@ -1,803 +1,906 @@
+        '<\'row\'<\'col-sm-12\'rt>>' +
+        '<\'row\'<\'col-sm-5\'i><\'col-sm-7\'p>>',
+        buttons: [
+            'copy', 'excel', 'pdf'
+        ],
+        language: {
+            lengthMenu: 'Display _MENU_ documents',
+            info: 'Showing _START_ to _END_ of _TOTAL_ documents',
+            zeroRecords: 'No documents to display',
+            select: {
+                rows: ''
+            }
+        },
+        columnDefs: [
+            {width: '15', sortable: false, class: 'index', targets: 0},
+            {width: '150', type: 'natural', targets: [1, 2, 3]}
+            //				{width: "150", type: 'natural', targets: "_all"},
+        ],
+        order: [[1, 'asc']]
     })
-    .on('deselect', function (e, dt, type, indexes) {
-      // Get deselected rows as a jQuery object
-      var deselected_rows = table.rows(indexes).nodes().to$()
-      // Call the ajax function
-      disableRows(deselected_rows)
-      handleSelectButtons(table.rows().ids().length, table.rows({ selected: true }).ids().length)
-      document.getElementById('name').innerHTML = table.rows('.selected').data().length + ' active documents' // same as the other one
-      $('.fa-folder-open-o')[0].dataset.originalTitle = 'You have ' + table.rows('.selected').data().length + ' active document(s)'
-      if (table.rows('.selected').data().length == 0) {
-        $('#bttn-downloadSelectedDocs').hide()
-      }
+
+    var selectee = table.rows('.selected').data().length
+    // console.log($('.dataTables_info'));
+
+    /* table.on('page.dt', function() {
+      table.state.clear();
+      window.location.reload();
+    }); */
+
+    // Draw the index column
+    table.on('order.dt search.dt', function () {
+        table
+            .column(0, {
+                    search: 'applied',
+                    order: 'applied'
+                }
+            )
+            .nodes()
+            .each(function (cell, i) {
+                cell.innerHTML = i + 1
+            })
+    })
+        .draw()
+
+    // Make all columns searchable
+    table.on('order.dt search.dt', function () {
+        table
+            .column(1, {
+                    search: 'applied',
+                    order: 'applied'
+                }
+            )
+            .column(2, {
+                    search: 'applied',
+                    order: 'applied'
+                }
+            )
+            .column(3, {
+                    search: 'applied',
+                    order: 'applied'
+                }
+            )
     })
+        .nodes()
+        .draw()
+
+    /* Add all rows with .selected to the DataTables activeRows array
+     It does not appear that this array needs to be maintained after
+     initialisation, but the code to do so is commented out for de-bugging. */
+    activeRows = []
+    $('tbody tr').each(function (index) {
+        if ($(this).hasClass('selected')) {
+            i = ':eq(' + index + ')'
+            table.rows(i).select()
```

Comment:
```js
table.row(`:eq(${index})`).select()
```
You don't really need the `i`

Also, add documents I don't understand what these lines are talking about